src/lib/api-client.js

```diff
@@ -86,6 +86,7 @@ export function checkOutItems(studentId, itemAddresses){
             Dispatcher.handleAction('OVERRIDE_REQUIRED');
         }else if(error === 'Invalid Admin'){
             Dispatcher.handleAction('CLEAR_ADMIN_CODE');
+            Dispatcher.handleAction('INVALID_CODE');
```

Comment:
You could also keep `CLEAR_ADMIN_CODE` and have the toast store subscribe to that, but `INVALID_CODE` seems more descriptive.